ddp-workspace/projects/study-builder-ui/src/app/components/text-question-editor/text-question-editor.component.ts

```diff
@@ -26,41 +27,57 @@ export class TextQuestionEditorComponent implements OnInit, OnDestroy {
         inputType: [''],
         stableId: [''],
         prompt: [''],
-        placeholder: [''],
-        required: [false]
+        placeholder: ['']
     });
-
+    initialValidators: RuleDef[];
     private sub: Subscription;
 
-    constructor(private fb: FormBuilder, private config: ConfigurationService) {
-    }
+    constructor(
+        private fb: FormBuilder,
+        private config: ConfigurationService
+    ) {}
 
     ngOnInit(): void {
         const updateFormPipe = this.questionBlockSubject.pipe(
             filter(block => !!block),
             map(block => block.question),
-            tap(question => this.updateForm(question as TextQuestionDef))
+            tap(question => {
+                this.initValidators(question);
+                this.updateForm(question as TextQuestionDef);
+            })
         );
         const updateQuestionPipe = this.formGroup.valueChanges.pipe(
             map(formData => this.updateQuestion(formData)),
             tap(() => this.questionBlockChanged.emit(this.questionBlockSubject.getValue()))
         );
-        this.sub = merge(updateFormPipe, updateQuestionPipe).subscribe();
+        this.sub = merge(updateFormPipe, updateQuestionPipe).subscribe({});
```

Comment:
I'm curious on this. Adding curly braces at the end of the subscribe has any impact?

Reply:
It was a linter warning about [deprecated subscribe-arguments](https://rxjs.dev/deprecations/subscribe-arguments).

Reply:
@josemanuel167 New rxjs in Angular 13 wants an object for .subscribe argument. @alextanlan had to fix those EVERYWHERE!